### PR TITLE
chore: adhere to style guide in `OutputBisectionGame` for immutables

### PR DESCRIPTION
**Pull request by clabby**
## Overview

Adds `camelCase` getters for `immutable` variables in the `OutputBisectionGame` contract and moves the `immutables` to `internal` visibility. Ugly PR, but necessary for style guide adherence.

**Metadata**
closes https://github.com/ethereum-optimism/client-pod/issues/258

### CODE REVIEW

**Comment by clabby** [comment]
Current dependencies on/for this PR:
* `develop`
  * **PR #8656** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/8656?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/8656?utm_source=stack-comment).

File: op-e2e/e2eutils/disputegame/helper.go [hunk 1]
```diff
@@ -34,11 +34,13 @@ import (
...
 )
 
-const alphabetGameType uint8 = 255
-const cannonGameType uint8 = 0
-const outputCannonGameType uint8 = 1
-const outputAlphabetGameType uint8 = 254
-const alphabetGameDepth = 4
+const (
+	alphabetGameType       uint8 = 255
+	cannonGameType         uint8 = 0
```
Comment (Inphi) [inline op-e2e/e2eutils/disputegame/helper.go:39]: nit: you can omit the type in a const decl block for all statements except the first one. The other decls will have the same type as the first.

**Review by Inphi — APPROVED** [review]
It's almost time!

File: op-e2e/e2eutils/disputegame/helper.go (hunk 1 above)
Comment (clabby) [reply op-e2e/e2eutils/disputegame/helper.go:39]: Ah this actually doesn't work - go lint complains about it.

**Comment by tynes** [comment]
If we upgraded the contracts on testnet then we could delete a bunch of the code in this PR right?

**Review by tynes — APPROVED** [review]

**Pull request merged**